README.md

```diff
@@ -79,12 +79,12 @@ If you want to use `ripple-lib` with React Native you will need to have some of
   ...
   ```
 
-### Using ripple-lib with Deno
+### Using xrpl.js with Deno
 
-Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `ripple-lib` with Deno:
+Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `xrpl.js` with Deno:
 
 ```javascript
-import ripple from 'https://dev.jspm.io/npm:ripple-lib';
+import ripple from 'https://dev.jspm.io/npm:xrpl.js';
 
 (async () => {
   const api = new (ripple as any).RippleAPI({ server: 'wss://s.altnet.rippletest.net:51233' });
```

Comment:
I assume we're not worried about changing sample code for now.

Reply:
I was not concerned about it... I also didn't change URLs or anything like that.